README.md

```diff
@@ -375,6 +385,11 @@ kubectl delete -f manifests
 istioctl delete -f manifests/<filename>.yaml #Replace <filename> with the rule/policy file name you want to delete.
 ```
 
+* If you have trouble with the maven build, your maven might have some dependency conflicts. Therefore, you need to purge your dependencies by running the following commands 
+```shell
+mvn clean package dependency:purge-local-repository
```

Comment:
This  worked for me `mvn dependency:purge-local-repository`. Had trouble with `mvn clean package dependency:purge-local-repository`